test_scripts/API/SetMediaClockTimer/ATF_HP_SetMediaClockTimer_With_Valid_enableSeek_Parameter.lua

```diff
@@ -0,0 +1,130 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [GENIVI] SetMediaClockTimer: SDL must support new parameter "enableSeek" and transfer OnSeekMediaClockTimer notification from HMI to mobile app
+--
+-- Description:
+-- SDL must support and process new "enableSeek" param at SetMediaClockTimer
+-- In case "enableSeek" is true -> SDL expects OnSeekMediaClockTimer from HMI and trabsfer this notification to mobile app
```

Comment:
@OVikhrov, description tells : "SDL expects OnSeekMediaClockTimer" - but this expectation is missed in script. update description or test in script to contain OnSeekMediaClockTimer